<launch>
  <node pkg="jetbot_picam" type="cam_pi.py" name="cam_pi" />
  <group ns="goalpost">
    <node pkg="jetbot_picam" type="color_filter.py" name="goalpost_detector" output="screen">
      <param name="MODE" value="2"/>
      <param name="lower_h" value="90"/>
      <param name="lower_s" value="90"/>
      <param name="lower_v" value="50"/>
      <param name="higher_h" value="160"/>
      <param name="higher_s" value="255"/>
      <param name="higher_v" value="200"/>
    </node>
  </group>

  <group ns="ball">
    <node pkg="jetbot_picam" type="color_filter.py" name="ball_detector" output="screen">
      <param name="MODE" value="1"/>
      <param name="lower_h" value="0"/>
      <param name="lower_s" value="100"/>
      <param name="lower_v" value="100"/>
      <param name="higher_h" value="15"/>
      <param name="higher_s" value="255"/>
      <param name="higher_v" value="255"/>
      <param name="lower_h_b" value="165"/>
      <param name="lower_s_b" value="100"/>
      <param name="lower_v_b" value="100"/>
      <param name="higher_h_b" value="180"/>
      <param name="higher_s_b" value="255"/>
      <param name="higher_v_b" value="255"/>
    </node>
  </group>
  <node pkg="jetbot_picam" type="jetbot_soccer.py" name="jetbot_soccer" output="screen"/>
</launch>
